lib/cdo/contact_rollups_validation.rb

```diff
@@ -344,4 +345,26 @@ def self.log(s)
     # puts s unless Rails.env.test?
     CDO.log.info s
   end
+
+  private
+
+  def initialize_connection_to_database_clone
```

Comment:
Reduce duplication with `initialize_connections_to_database_clone` in `contact_rollups.rb`?

Reply:
Done!